fixieai/cli/agent/commands.py

```diff
@@ -104,6 +106,38 @@ def list_agents(ctx, verbose):
             click.echo()
 
 
+@agent.command("show", help="Show an agent.")
+@click.argument("agent_id")
+@click.pass_context
+def show_agent(ctx, agent_id: str):
+    client = ctx.obj.client
+    agent = client.get_agent(agent_id)
+    if not agent.agent_id:
+        click.echo(f"Agent {agent_id} not found.")
+        return
+    click.secho(f"{agent.agent_id}", fg="green", nl=False)
+    click.echo(f": {agent.name}")
+    click.echo(f"Owner: {agent.owner}")
+    click.echo(f"Description: {agent.description}")
+    click.echo(f"More info URL: {agent.more_info_url}")
+    click.echo(f"Published: {agent.published}")
+    click.echo(f"Func URL: {agent.func_url}")
+    click.echo(f"Query URL: {agent.query_url}")
```

Comment:
should we hide fields that are None? I hate to expose system innards unnecessarily.

Reply:
If you don't mind I'd like to fix this later, trying to get this done quickly and bouncing between a lot of things...